.github/workflows/ci.yaml

```diff
@@ -576,6 +576,8 @@ jobs:
         run: make kind-import
 
       # ---- Install Test Dependencies ----
+      - name: Update NPM
+        run: npm i -g npm@${{ env.NPM_VERSION }}
```

Comment:
Is it a good idea to upgrade global npm? I use nvm. I suggest to use nvm. We already have a .nvmrc in the repo.

Reply:
I file a new issue about this.

Reply:
https://github.com/secureCodeBox/secureCodeBox/issues/1581